wasm-engines/fizzy.sh

```diff
@@ -13,7 +13,7 @@ echo $'\n' >> testcase.inputs   # 6. Expected memory
 # Copy wasm file
 cp $2 ./testcase.wasm
 
-/engines/fizzy/build/bin/fizzy-bench --benchmark_color=false ./
+/engines/fizzy/build/bin/fizzy-bench --benchmark_color=false ./ | sed 's/\([0-9][0-9]*\)\s\([nm]s\)/\1\2/g'
```

Comment:
It can also output CSV or JSON:`           [--benchmark_out_format=<json|console|csv>]`

Reply:
I tried CSV and JSON, the unit is separated from the value, the regex in the python file expect it to be together and that regex is used for all engines. I added that sed command to put the number and unit together (no spaces in between).